lib/help.js

```diff
@@ -34,7 +34,7 @@ class Help {
     const description = this._program.description() ? '- ' + this._program.description() : '';
     let help = `
    ${chalk.cyan(this._program.name() || this._program.bin())} ${chalk.dim(this._program.version())} ${description}
-     
+
```

Comment:
The additional whitespace was removed by an extension I run. I hope this won't be a problem.